src/index-bin.ts

```diff
@@ -16,8 +16,8 @@ function mkdirRecursive(dir: string) {
   let sep = '/';
 
   let segments = dir.split(sep);
-  let current = '';
-  let i = 0;
+  let current = segments[0];
+  let i = 1;
 
   while (i < segments.length) {
     current = current + sep + segments[i];
```

Comment:
Assuming we want a directory `a/b/c` the line 23 would first try to create `a/a`. Is that what you wanted to do?

Reply:
Oops, I did not see the line 20, dismiss my comment